.github/workflows/functional-tests.yml

```diff
@@ -0,0 +1,81 @@
+name: Functional tests
+on:
+  pull_request:
+    branches: [1.*]
+jobs:
+  simpletest:
+    name: Simpletest batches
+    runs-on: ubuntu-18.04
```

Comment:
Can we bump this to 20.04, the most recent Ubuntu LTS release?

Reply:
No problem, but then I'd have to install a different Database, because 20.04 ships with MySQL8 - which Backdrop doesn't fully support yet.

Reply:
> we can tackle this in a follow-up

Phew, maybe it was just bad timing - being in a hurry is never helpful. :wink: I'm sure we can figure out what's so different in U20 or M8, that it doesn't work.